mmdet/models/detectors/single_stage.py

```diff
@@ -111,7 +111,10 @@ def simple_test(self, img, img_metas, rescale=False, postprocess=True):
             bbox_results = \
                 self.bbox_head.get_bboxes(*outs, img_metas, self.test_cfg, False)
         if torch.onnx.is_in_onnx_export() or is_in_nncf_tracing():
-            return bbox_results[0]
+            feature_vector = get_feature_vector(x)
+            saliency_map = get_saliency_map(x[0])
```

Comment:
Could you explain the meaning of x[0]? It seems we are using the output from the 1st layer of backbone as saliency map. Is that correct? What happens if we choose to use x[-1]?

Reply:
That would be better in visual quality I guess but let's go simple w/ the last layer output as it has most high-level abstract semantic. Image size would not matter as it will be resized in OTE task.